.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+        
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
+        poetry update
+        
+    - name: Set up .env file
+      run: |
+        DBNAME="AnswerkingDBTest"
+        DBUSER="root"
+        DBPWD="UnsafePassword098"
+        DBHOST="127.0.0.1"
+        DBPORT="3306"
+        DBSECRET=`openssl rand -base64 32`
+        echo -e "DATABASE_NAME=$DBNAME\nDATABASE_USER=$DBUSER\nDATABASE_PASS=$DBPWD\nDATABASE_HOST=$DBHOST\nDATABASE_PORT=$DBPORT\nSECRET_KEY=$DBSECRET" > answerking/.env
+
+    - name: Shutdown Ubuntu MySQL (SUDO)
+      run: sudo service mysql stop
+        
+    - name: Set up MySQL
+      uses: mirromutth/mysql-action@v1.1
+      with: 
+        host port: 3306
+        mysql database: 'AnswerKingBDTest'
+        mysql root password: "UnsafePassword098"
+  
+  
+  test:
+    runs-on: ubuntu-latest
```

Comment:
My guess is that it's failing because you are redefining the environment and hence you don't have the dependencies installed